functions/library/KOBOFetcher/components/dataParser.ts

```diff
@@ -22,7 +22,7 @@ const dataParser = (data: Array<any>, survey: Survey): Array<Attribute> => {
       },
       properties: {
         KOBO_uuid: item._uuid,
-        createdDate: item._submission_time,
+        date: item._submission_time,
```

Comment:
I'm not sure it's the best solution, I would think, it's better to modify frontend, as only 'date' with also 'updatedDate' field might be confusing. Also 'createdDate' is only first date when the data is submitted and as a user, I would be expect date of last change, since when are this data up-to-date